examples/agent_motion_prediction/agent_motion_prediction.ipynb

```diff
@@ -231,7 +228,11 @@
     "In this example, we simulate this pipeline by running `chop_dataset` on the validation set. The function stores:\n",
     "- a new chopped `.zarr` dataset, in which each scene has only the first 100 frames;\n",
     "- a numpy mask array where only valid agents in the 100th frame are True;\n",
-    "- a ground-truth file with the future coordinates of those agents;"
+    "- a ground-truth file with the future coordinates of those agents;\n",
+    "\n",
+    "Please note how `Num Frames==(Num Scenes)*num_frames_to_chop`. \n",
```

Comment:
maybe be consistent with notation.. either num_frames=num_scenes*num_frames_to_chop.  or  `Num Frames==(Num Scenes)*Num Frames To Chop  (minor)

Reply:
Also, maybe remove brackets around Num Scenes either way

Reply:
Yes and maybe just use one = spaces around, as in 'speaking' instead of code.